app/dashboard/views.py

```diff
@@ -731,6 +731,50 @@ def onboard(request, flow):
     return TemplateResponse(request, 'ftux/onboard.html', params)
 
 
+def users_directory(request):
+    """Handle displaying users directory page."""
+
+    params = {
+        'active': 'users',
+        'title': 'Users',
+        'meta_title': "",
+        'meta_description': ""
+    }
+    return TemplateResponse(request, 'dashboard/users.html', params)
+
+
+@require_GET
+def users_fetch(request):
+    """Handle displaying users."""
+    q = request.GET.get('search','')
```

Comment:
add a space before the final string here like `q = request.GET.get('search', '')`